BuildAndTest.proj

```diff
@@ -5,13 +5,18 @@
   
   <PropertyGroup>
     <SolutionFile>$(MSBuildThisFileDirectory)RoslynAnalyzers.sln</SolutionFile>
-    <PackagingProject>$(MSBuildThisFileDirectory)\src\Packaging\Packaging.proj</PackagingProject>
+    <PackagingProject>$(MSBuildThisFileDirectory)src\Packaging\Packaging.proj</PackagingProject>
+    <BinariesPath>$(MSBuildThisFileDirectory)Binaries\$(Configuration)</BinariesPath>
     <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>
     <IncludePattern Condition="'$(IncludePattern)' == ''">*.Test*.dll</IncludePattern>
-
+    
     <!-- Emit XML in order to get structured test results -->
     <RunTestArgs>-noshadow -parallel all -xml Binaries\$(Configuration)\TestResults.xml</RunTestArgs>
 
+    <!-- Arguments to the SignTool -->
+    <SignRoslynArgs>-msbuildPath "$(MSBuildBinPath)\msbuild.exe"</SignRoslynArgs>
```

Comment:
I sincerely do not like the implicit config path strategy that signtool uses. Last time I had to go and make changes, finding out where the config was located, what config it was using (roslyn had 2), and why it was magically finding it wasted a bunch of my time. We can keep the config where it is, but lets please actually pass the argument to signtool with `-config <path to config>` to reduce the magic here.